src/utils/winner_showcase.rs

```diff
@@ -102,15 +103,15 @@ pub async fn display_winner(http: Arc<Http>, db: impl AsRef<Database>, offset: i
                             Ok(m) => {
                                 e.field(
                                     format!("Sijalla {}.", ranking),
-                                    format!("{}, {} viestiä ({:.1} %)", m, msg_count, msg_percent),
+                                    format!("{}, {} viestiä ({:,1} %)", m, msg_count, msg_percent),
```

Comment:
This doesn't work and causes a compiler error, it needs to be changed back to '.'
If a comma is wanted here, @RoyTakanen has suggested an alternative (although a bit more involved) solution
in the comments of this pull request. I assume this will be fixed soon.